store/sqlstore/status_store.go

```diff
@@ -99,6 +102,89 @@ func (s SqlStatusStore) GetByIds(userIds []string) ([]*model.Status, error) {
 	return statuses, nil
 }
 
+// MySql doesn't has support for RETURNING clause or any euivalent so use transaction
+func (s SqlStatusStore) updateExpiredStatuses(t *gorp.Transaction) ([]*model.Status, error) {
+	selectQuery, selectParams, _ := s.getQueryBuilder().
+		Select("UserId, Status").
+		From("Status").
+		Where(
+			sq.Eq{"Status": model.STATUS_DND},
+			sq.Gt{"DNDEndTimeUnix": -1},
+			sq.LtOrEq{"DNDEndTimeUnix": time.Now().UTC().Unix()},
```

Comment:
I would get the `time.Now().UTC().Unix()` as a variable to make the both queries completly identical.